api/src/repositories/code-repository.ts

```diff
@@ -482,13 +484,32 @@ export class CodeRepository extends BaseRepository {
         name,
         description
       FROM device_make
-      WHERE
-        record_effective_date IS NOT NULL
```

Comment:
small note: Some of the `device_make` seed data intentionally had no `record_effective_date`. ie: `teleonics` - which is a telemetry vendor but not supported by SIMS / BCTW.

My thinking was I wanted the pick lists in the frontend to be populated with the exhaustive list of vendors that map 1 to 1 to the true vendor telemetry tables.

Reply:
Gotcha, will fix